cal/cal/models.py

```diff
@@ -28,6 +28,7 @@ class Profile(models.Model):
     main_calendar = models.ForeignKey("GCalendar", null=True)
     authed = models.BooleanField(default=False, help_text="If the user's oauth credentials are currently valid")
     analysis_start = models.DateTimeField(null=True, blank=True, help_text="When the analysis of the user's calendar will start")
+    first_time_authed = models.BooleanField(default=False, help_text="If it's the first time the user's oauth credentials are authenticated")
```

Comment:
You should commit your migrations too. Other people on the project should only need to run `python manage.py migrate`.